spec/sum.js

```diff
@@ -1,4 +1,4 @@
-function sum(a, b) {
-  return a + b;
+function sum(x, y) {
+  return x + y;
 }
 module.exports = sum;
```

Comment:
@esme 
@ctong5 

I don't see a newline at the end of this file, which is part of the Airbnb Style Guide--is ESLint setup? would recommend fixing in all files, and ensuring that your code is passing a linter before committing!